app/src/test/java/nl/tudelft/b_b_w/controller/BlockControllerUnitTest.java

```diff
@@ -52,7 +53,7 @@
     public void setUp() {
         this.bc = new BlockController(RuntimeEnvironment.application);
         this._block = BlockFactory.getBlock(TYPE_BLOCK, owner, sequenceNumber, ownHash,
```

Comment:
You should not be able to give a sequencenumber to the constructor upon creating a block